packages/tasks/src/local-apps.ts

```diff
@@ -48,20 +48,25 @@ function isGgufModel(model: ModelData) {
 
 const snippetLlamacpp = (model: ModelData): string[] => {
 	return [
-		`## Install llama.cpp via brew
+		`# Option 1: use llama.cpp with brew
 brew install llama.cpp
 
-## or from source with curl support
-## see llama.cpp README for compilation flags to optimize for your hardware
-git clone https://github.com/ggerganov/llama.cpp
-cd llama.cpp
-LLAMA_CURL=1 make
-`,
-		`## Load and run the model
+# Load and run the model
```

Comment:
```suggestion
# Build llama.cpp then load and run the model
```

Reply:
No it was better before can you revert @mishig25 ?

Reply:
https://github.com/huggingface/huggingface.js/pull/737/commits/1ef3a5e8a5577e7c1eb73d546ff0388794af43aa